libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -6,6 +6,7 @@
       type="search"
       placeholder="Search for books to add to your reading list"
       formControlName="term"
+      (input)="searchBooks()"
```

Comment:
We do not want to be using template-driven techniques, but rather Reactive forms.

Reply:
Removed the reactivity from the template